sdk/lib/rtl/actctx.c

```diff
@@ -2403,19 +2364,22 @@ static void parse_file_elem( xmlbuf_t* xmlbuf, struct assembly* assembly,
 
     while (next_xml_attr(xmlbuf, &attr, &end))
     {
-        if (xml_attr_cmp(&attr, g_nameW))
+        if (xml_attr_cmp(&attr, L"name"))
         {
             if (!(dll->name = xmlstrdupW(&attr.value))) set_error( xmlbuf );
             TRACE("name=%s\n", debugstr_xmlstr(&attr.value));
         }
-        else if (xml_attr_cmp(&attr, hashW))
+        else if (xml_attr_cmp(&attr, L"loadFrom"))
+        {
+            if (!(dll->load_from = xmlstrdupW(&attr.value))) set_error( xmlbuf );
```

Comment:
neat, we were lacking this undocumented thing indeed, now only to use that while loading dlls!

Reply:
Would this help with parts of [CORE-19707](https://jira.reactos.org/browse/CORE-19707)? It just sets the ACTIVATION_CONTEXT_DATA_DLL_REDIRECTION_PATH_EXPAND flag but where does it expand?

Reply:
This is the `loadFrom` thing, that tells sxs to ignore the default load paths.
I don't think it is related to the `ACTIVATION_CONTEXT_DATA_DLL_REDIRECTION_PATH_EXPAND` flag, but I am not certain.

This thread suggests that they did indeed apply this as mitigation against dll proxying attacks:
https://groups.google.com/g/innosetup/c/IgoRx-i2LeU

Reply:
The flag gets set if the loadfrom path contains a %.

Reply:
right,
but considering we completely ignored `loadFrom` until now I doubt we would do anything with one of it's flags.